addons/mail/static/src/models/clock/clock.js

```diff
@@ -0,0 +1,75 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many } from '@mail/model/model_field';
+import { OnChange } from '@mail/model/model_onchange';
+
+/**
+ * Models a record that provides the current date, updated at a given frequency.
+ */
+registerModel({
+    name: 'Clock',
+    identifyingFields: ['frequency'],
+    lifecycleHooks: {
+        _willDelete() {
+            this.messaging.browser.clearInterval(this.tickInterval);
+        },
+    },
+    recordMethods: {
+        /**
+         * @private
+         * @returns {integer}
+         */
+        _computeTickInterval() {
+            return this.messaging.browser.setInterval(this._onInterval, this.frequency);
+        },
+        /**
+         * @private
+         */
+        _onChangeWatchers() {
+            if (this.watchers.length === 0) {
+                this.delete();
+            }
+        },
+        /**
+         * @private
+         */
+        _onInterval() {
+            this.update({ date: new Date() });
+        },
+    },
+    fields: {
+        /**
+         * A Date object set at the current date, updated at every tick.
+         */
+        date: attr({
```

Comment:
note: confusing name

should be browser.date/web.date?

but it's actually a date + time (even the browser one is badly named)